src/Microsoft.AspNetCore.Authentication.Abstractions/AuthenticateResult.cs

```diff
@@ -31,12 +31,17 @@ public class AuthenticateResult
         /// <summary>
         /// Additional state values for the authentication session.
         /// </summary>
-        public AuthenticationProperties Properties => Ticket?.Properties;
+        public AuthenticationProperties Properties => Ticket?.Properties ?? Error?.Properties;
```

Comment:
Sure but the Error is just an exception + properties, can't you do the same with an ErrorResult (which also has Failure exception + Properties)?  Where's the relevant code in the associated PR today?